README.md

```diff
@@ -19,6 +24,78 @@ This will prompt for your password because it is
 using HTTPS on port 443 which is a privileged port.
 
 To change the owner (ex. root) of a table:
+```bash
 psql -d launchpad
 alter table {table-name} owner to {new-owner}
 press ctrl-d
+```
+
+## Environment Variables
+
+The following environment variables are used for the database and the API:
+
+```bash
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+POSTGRES_HOST=db
+POSTGRES_DB=launchpad
+POSTGRES_PORT=5432
+POSTGRES_POOL=10
+POSTGRES_TIMEOUT=30000
+```
+
+`POSTGRES_USER` and `POSTGRES_PASSWORD` do not have defaults so you will need to set those in your environment.
+Add them to your `.bash_profile`:
+```bash
+export POSTGRES_USER=user
+export POSTGRES_PASSWORD=pass
+```
+
+## Docker Compose
+
+To install Docker follow the instructions here: [Installing Docker](https://docs.docker.com/docker-for-mac/install/)
+*NOTE: You could use `brew` to install docker, but that won't give you `docker-compose`, `docker-machine` or the other docker tools.*
+
+Once Docker is installed and running you can proceed with standing up a local stack:
+In a shell run:
+```bash
+dup
+```
+*NOTE: `dup` is an alias. To review the actual command see the `aliases` below*
+
+This will run docker-compose to stand up two containers: `api` and `db`
+*NOTE: These containers know how to talk to each other via docker networking.*
+
+We also **expose the ports** for these containers in the host machine, so you can test locally:
+In a browser visit: https://localhost/project
+
+Here are some useful docker aliases for this project:
+```bash
+# Usage: ddown
+alias ddown="docker-compose -p launchpad down --volumes"
+
+# Usage: dkimages
+alias dkimages="docker rmi $(docker images -f 'dangling=true' -q) || echo No images to kill"
+
+# Usage: dup
+alias dup="dkimages && ddown && docker-compose -p launchpad up -d --force-recreate --build"
+
+# Usage: dexec db /bin/sh
```

Comment:
I put a note in there that you could connect pgadmin or other gui's to the container.